src/Navbar.jsx

```diff
@@ -24,7 +26,34 @@ export const StyledLink = styled(Link)`
   color: white;
 `;
 
-const Navbar = () => {
+const Navbar = ({ token, deleteToken }) => {
+  const history = useHistory();
+  const logout = async () => {
+    try {
+      await logoutUser(token);
+      deleteToken();
+      history.push("/");
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const changeLoginStatus = () => {
```

Comment:
would probably prefer to see this inline or as a separate component, this seems halfway between.